### PR TITLE
Ck Editor Watchdog integration

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

File: packages/lesswrong/components/editor/ReactCKEditor.jsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+          
+          if ( this.props.onInit ) {
+            this.props.onInit( editor );
+          }
+          
+          const modelDocument = editor.model.document;
+          const viewDocument = editor.editing.view.document;
+          
+          modelDocument.on( 'change:data', event => {
+            /* istanbul ignore else */
```
Comment (jimrandomh) [inline packages/lesswrong/components/editor/ReactCKEditor.jsx]: Is this pasted from somewhere?

**Comment by jimrandomh** [comment]
Looks pretty high context; we should have a conversation about this one.

File: packages/lesswrong/components/editor/ReactCKEditor.jsx (hunk 1 above)
Comment (Discordius) [reply packages/lesswrong/components/editor/ReactCKEditor.jsx]: Ah, yes. Let me add the source (it's the component we were using anyways, just copy pasted and slightly modified)

File: packages/lesswrong/components/editor/ReactCKEditor.jsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Watchdog } from '@lesswrong/lesswrong-editor'
+
```
Comment (Discordius) [inline packages/lesswrong/components/editor/ReactCKEditor.jsx]:
```suggestion
Copied from: https://github.com/ckeditor/ckeditor5-react/blob/master/src/ckeditor.jsx
```

File: packages/lesswrong/components/editor/ReactCKEditor.jsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Watchdog } from '@lesswrong/lesswrong-editor'
+Copied from: https://github.com/ckeditor/ckeditor5-react/blob/master/src/ckeditor.jsx
```
Comment (Discordius) [inline packages/lesswrong/components/editor/ReactCKEditor.jsx]:
```suggestion
// Copied from: https://github.com/ckeditor/ckeditor5-react/blob/master/src/ckeditor.jsx
```

File: packages/lesswrong/components/editor/ReactCKEditor.jsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Watchdog } from '@lesswrong/lesswrong-editor'
+// Copied from: https://github.com/ckeditor/ckeditor5-react/blob/master/src/ckeditor.jsx
```
Comment (Discordius) [inline packages/lesswrong/components/editor/ReactCKEditor.jsx]:
```suggestion
// Copied from and modified: https://github.com/ckeditor/ckeditor5-react/blob/master/src/ckeditor.jsx
```

**Pull request merged**